src/util/utils.ts

```diff
@@ -60,3 +60,52 @@ export const formatImageURL = (
   return `${url}.${format ||
     (url.includes("/a_") ? "gif" : "jpg")}?size=${size}`;
 };
+
+export function camelToSnakeCase(str: string) {
+  return str.replace(/ID|[A-Z]/g, (s) => {
+    if (s === "ID") return "_id";
+    return `_${s.toLowerCase()}`;
+  });
+}
+
+export function snakeToCamelCase(s: string) {
```

Comment:
```suggestion
function snakeToCamelCase(s: string) {
```
Do not export private functions